integration/utils.go

```diff
@@ -19,7 +19,7 @@ const decompressDirsBinary = "./decompress-dirs.sh"
 
 const testFQIN = "docker://quay.io/libpod/busybox" // tag left off on purpose, some tests need to add a special one
 const testFQIN64 = "docker://quay.io/libpod/busybox:amd64"
-const testFQINMultiLayer = "docker://quay.io/libpod/alpine_nginx:master" // multi-layer
+const testFQINMultiLayer = "docker://quay.io/libpod/alpine_nginx:latest" // multi-layer
```

Comment:
(Note to self: I have verified that the two tags currently point at exactly the same image.

The original reference comes from 285a5cb6a00a4ce0058fffed18c2df45d13cd171 , which doesn’t suggest there’s any automated maintenance process for this repo that needs updating.)